### PR TITLE
Make things scalable

### PR DESCRIPTION
**Pull request by DeveloperKhan**
Containerizing backend server

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,11 +22,13 @@
   "type": "module",
   "types": "./types/index.ts",
   "dependencies": {
+    "@types/redis": "^2.8.28",
```
Comment (DeveloperKhan) [inline package.json:25]: move this to dev dependencies

File: types/package.json [hunk 2]
```diff
@@ -15,6 +15,9 @@
     "type": "git",
     "url": "git+https://github.com/DeveloperKhan/pogo-web-backend.git"
   },
+  "dependencies": {
+    "@types/redis": "^2.8.28"
```
Comment (DeveloperKhan) [inline types/package.json:19]: move this also to devDependencies

**Review by DeveloperKhan — COMMENTED** [review]
Looks good so far! Ty

**Review by FlorianKutz — COMMENTED** [review]
I mean, I made these changes myself. It's not very efficient if I need to do the review.

**Review by DeveloperKhan — COMMENTED** [review]
Looks good

**Pull request merged**